.github/workflows/sonar.yml

```diff
@@ -60,4 +61,10 @@ jobs:
           .\.sonar\scanner\dotnet-sonarscanner begin /k:"${{ secrets.SONAR_PROJECT_KEY }}" /o:"${{ secrets.SONAR_CASBIN_ORG_NAME }}" /d:sonar.login="${{ secrets.SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io" /d:sonar.cs.opencover.reportsPaths="${{ GITHUB.WORKSPACE }}\**\coverage*.opencover.xml" /d:sonar.cs.vstest.reportsPaths="${{ GITHUB.WORKSPACE }}\**\*.trx" /d:sonar.verbose=true
           dotnet build -c Release
           dotnet test -c Release --logger trx -r test-results --collect:"XPlat Code Coverage" /p:CollectCoverage=true /p:CoverletOutputFormat=opencover -f net5 -- DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=json,cobertura,lcov,teamcity,opencover
-          .\.sonar\scanner\dotnet-sonarscanner end /d:sonar.login="${{ secrets.SONAR_TOKEN }}"
\ No newline at end of file
+          .\.sonar\scanner\dotnet-sonarscanner end /d:sonar.login="${{ secrets.SONAR_TOKEN }}"
+      #- name: Remove sonarcloud trigger label
```

Comment:
I can, but given the current status, I don't think they will work as expected. There's very little documentation on this, so I would to close this PR, get the external PRs going through as expected, and then automate the removal of the label.
Can we close as is?, since its not a bug or a gross-bad-practice

Reply:
We can try to implement this later.